3rdparty/fastcv/CMakeLists.txt

```diff
@@ -24,8 +24,6 @@ if(HAVE_FASTCV)
     ocv_install_target(fastcv_hal EXPORT OpenCVModules ARCHIVE DESTINATION ${OPENCV_3P_LIB_INSTALL_PATH} COMPONENT dev)
   endif()
 
-  ocv_install_3rdparty_licenses(FastCV "${OpenCV_BINARY_DIR}/3rdparty/fastcv/LICENSE")
```

Comment:
Perhaps it would be better to restore the LICENSE file instead with at least short license description:
```
Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
SPDX-License-Identifier: Apache-2.0
```

Reply:
The command moved to cmake/OpenCVFindLibsPerf.cmake. I just fixed the case, when CMake is pointed to some external instance of FastCV, e.g.  build root for Linux. The license file is not there.